thingif/src/main/java/com/kii/thingif/query/AggregatedResult.java

```diff
@@ -0,0 +1,47 @@
+package com.kii.thingif.query;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.kii.thingif.TargetState;
+
+import java.util.Date;
+import java.util.List;
+
+public class AggregatedResult<T extends Number> {
+    private @NonNull Date rangeFrom;
+    private @NonNull Date rangeTo;
```

Comment:
I refactor `TimeRange` in commit: https://github.com/KiiPlatform/thing-if-AndroidSDK/pull/170/commits/24bf1a0ecdc0535accf4deea3221477bbad96192